src/toggleUICollapsed.jsx

```diff
@@ -1,24 +1,34 @@
-function toggleUICollapsed(context, _event, actionMeta) {
-  if (!context.uiContainer) {
+function toggleUICollapsed(
+  uiContainer,
+  uiCollapsed,
+  selectedName,
+  service,
+  use2D,
+  planeUIGroup,
+  viewMode,
+  mainPlaneUIGroup,
+  _event,
+  actionMeta
+) {
```

Comment:
For many argument, can do the JS's version of keyword arguments with object destructing, as argument order is easy to mess up.
```
function toggleUICollapsed({
  uiContainer,
  uiCollapsed,
  selectedName,
  service,
  use2D,
  planeUIGroup,
  viewMode,
  mainPlaneUIGroup,
  _event,
  actionMeta
})
```

But better to think about how to reduce args.

Reply:
Oh interesting! Thanks for letting me know this is possible.